app/main/views/applications.py

```diff
@@ -76,6 +76,33 @@ def update_application(application_id):
     return jsonify(application=application.serializable), 200
 
 
+@main.route('/applications/<int:application_id>/admin', methods=['PUT'])
+def update_application_admin(application_id):
+    application_json = get_application_json()
+
+    application = Application.query.get(application_id)
+    if application is None:
+        abort(404, "Application '{}' does not exist".format(application_id))
+
+    if application.status == 'submitted' or application.status == 'saved':
+        db.session.add(AuditEvent(
```

Comment:
Does this actually get committed to the DB? if so I assume it must get rolled up in a commit inside of `save_application` below?

Reply:
yep